plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/messages/ModelResources_ja.properties

```diff
@@ -76,32 +79,66 @@ model_jdbc_None = \u306A\u3057
 model_jdbc_precision = \u7CBE\u5EA6
 model_jdbc_Procedure = \u30D7\u30ED\u30FC\u30B8\u30E3
 model_jdbc_rename_object = \u30AA\u30D6\u30B8\u30A7\u30AF\u30C8\u306E\u540D\u524D\u3092\u5909\u66F4
-model_jdbc_repeatable_read = \u7E70\u308A\u8FD4\u3057\u8AAD\u307F\u8FBC\u3080
+model_jdbc_read_committed = READ COMMITTED
+model_jdbc_read_uncommitted = READ UNCOMMITTED
+model_jdbc_repeatable_read = REPEATABLE READ
 model_jdbc_save_to_file_ = \u30D5\u30A1\u30A4\u30EB\u306B\u4FDD\u5B58 ...
 model_jdbc_scale = \u30B9\u30B1\u30FC\u30EB
 model_jdbc_Schema = \u30B9\u30AD\u30FC\u30DE
+model_jdbc_Serializable = SERIALIZABLE
 model_jdbc_unknown = \u4E0D\u660E
 model_navigator_Description = \u8AAC\u660E
 model_navigator_load_ = \u8AAD\u307F\u8FBC\u3080 
 model_navigator_load_items_ = \u30A2\u30A4\u30C6\u30E0\u3092\u8AAD\u307F\u8FBC\u3080 ...
+model_navigator_Model_root = \u30E2\u30C7\u30EB\u30EB\u30FC\u30C8
 model_navigator_Name = \u540D\u524D
 model_navigator_Project = \u30D7\u30ED\u30B8\u30A7\u30AF\u30C8
+model_navigator_Root = \u30EB\u30FC\u30C8
+model_struct_Association = \u30A2\u30BD\u30B7\u30A8\u30FC\u30B7\u30E7\u30F3
+model_struct_Cascade = CASCADE
+model_struct_Check = \u30C1\u30A7\u30C3\u30AF\u5236\u7D04
+model_struct_Clustered = \u30AF\u30E9\u30B9\u30BF\u30FC\u5316
 model_struct_Foreign_Key = \u5916\u90E8\u30AD\u30FC
+model_struct_Hashed = \u30CF\u30C3\u30B7\u30E5
 model_struct_Index = \u30A4\u30F3\u30C7\u30C3\u30AF\u30B9
 model_struct_Inheritance = \u7D99\u627F
+model_struct_No_Action = NO ACTION
+model_struct_Not_NULL = NOT NULL
 model_struct_Other = \u305D\u306E\u4ED6
 model_struct_Primary_Key = \u4E3B\u30AD\u30FC
-model_struct_Restrict = \u5236\u9650
+model_struct_Restrict = RESTRICT
+model_struct_Set_Default = SET DEFAULT
+model_struct_Set_NULL = SET NULL
```

Comment:
IMHO, some of foreign-keys no-need translate, same with isolation-level translation.
I have written in English to make it clear that there is no need to translate.

Please see MySQL Japanese docs. These words were not translated.

> https://dev.mysql.com/doc/refman/5.6/ja/create-table-foreign-keys.html